From fa7e062962d17862032bc600bfcffd486addb063 Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Date: Wed, 17 Mar 2010 13:52:38 -0500
Subject: [PATCH 12/37] Use EXEEXT in relink rules for portable DDXs

On Cygwin and MinGW, executables use the .exe suffix.  Autoconf and
automake set EXEEXT on these platforms, and leave it empty on others
where no suffix is used.  $(EXEEXT) must be appended to executable names
in custom rules for portability:

http://www.gnu.org/software/automake/manual/html_node/EXEEXT.html

Signed-off-by: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Reviewed-by: Alan Coopersmith <alan.coopersmith@sun.com>
Reviewed-by: Julien Cristau <jcristau@debian.org>
---
 hw/dmx/Makefile.am   |    2 +-
 hw/vfb/Makefile.am   |    2 +-
 hw/xnest/Makefile.am |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git xorg-server-1.7.99.902/hw/dmx/Makefile.am xorg-server-1.7.99.902/hw/dmx/Makefile.am
index 3c59320..1e3e5da 100644
--- xorg-server-1.7.99.902/hw/dmx/Makefile.am
+++ xorg-server-1.7.99.902/hw/dmx/Makefile.am
@@ -114,4 +114,4 @@ SUFFIXES = .$(APP_MAN_SUFFIX) .man
 EXTRA_DIST = $(appman_PRE)
 
 relink:
-	rm -f Xdmx && $(MAKE) Xdmx
+	rm -f Xdmx$(EXEEXT) && $(MAKE) Xdmx$(EXEEXT)
diff --git xorg-server-1.7.99.902/hw/vfb/Makefile.am xorg-server-1.7.99.902/hw/vfb/Makefile.am
index c5b49a3..c6601c2 100644
--- xorg-server-1.7.99.902/hw/vfb/Makefile.am
+++ xorg-server-1.7.99.902/hw/vfb/Makefile.am
@@ -49,4 +49,4 @@ SUFFIXES += .$(APP_MAN_SUFFIX) .man
 EXTRA_DIST = Xvfb.man.pre
 
 relink:
-	rm -f Xvfb && $(MAKE) Xvfb
+	rm -f Xvfb$(EXEEXT) && $(MAKE) Xvfb$(EXEEXT)
diff --git xorg-server-1.7.99.902/hw/xnest/Makefile.am xorg-server-1.7.99.902/hw/xnest/Makefile.am
index f95aab1..3e97490 100644
--- xorg-server-1.7.99.902/hw/xnest/Makefile.am
+++ xorg-server-1.7.99.902/hw/xnest/Makefile.am
@@ -89,4 +89,4 @@ SUFFIXES += .$(APP_MAN_SUFFIX) .man
 	$(LN_S) $< $@
 
 relink:
-	rm -f Xnest && $(MAKE) Xnest
+	rm -f Xnest$(EXEEXT) && $(MAKE) Xnest$(EXEEXT)
-- 
1.7.0.2

